rust-runtime/aws-smithy-http-server-python/rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+edition = "2021"
+max_width = 120
+# Prevent carriage returns
+newline_style = "Unix"
```

Comment:
I'm kind of surprised we haven't had any issues with the `aws-smithy-server` crate failing lints where rustfmt formatted it properly according to this config (see https://github.com/awslabs/smithy-rs/blob/a0539e20b069a7de021c84521d8f3c7ba098ad6d/tools/ci-build/scripts/check-style-and-lints#L14-L13) since the default `max_width` is 100. Or maybe we have and I just haven't noticed.

I'm completely onboard with making these settings apply to all of our Rust crates if you want. We should probably be consistent about it though to keep CI simpler.

No need to block on this--just be aware that CI may give you some grief while it remains inconsistent.

Reply:
I am happy to remove this and make it consistent. I am a fan of 120 chars, but I am ok also with 100 :)

Reply:
I'm also a fan of 120, so either way